testfiles/CMakeLists.txt

```diff
@@ -735,6 +735,7 @@ add_simulation_test(IDF_FILE WCE_DoubleClear_BSDF.idf EPW_FILE USA_IL_Chicago-OH
 add_simulation_test(IDF_FILE WCE_Interior_VB_-45_deg.idf EPW_FILE USA_IL_Chicago-OHare.Intl.AP.725300_TMY3.epw)
 add_simulation_test(IDF_FILE _5ZoneAirCooled_annual.idf EPW_FILE USA_IL_Chicago-OHare.Intl.AP.725300_TMY3.epw ANNUAL_SIMULATION)
 add_simulation_test(IDF_FILE _5ZoneAirCooled_LeapYear_annual.idf EPW_FILE MadeUpLeapYear.epw ANNUAL_SIMULATION)
+add_simulation_test(IDF_FILE UnitarySystem_MultiSpeedDX_EconoStaging.idf EPW_FILE USA_IL_Chicago-OHare.Intl.AP.725300_TMY3.epw ANNUAL_SIMULATION)
```

Comment:
New example file which uses the new input in 4 different configurations:
- `Coil:Cooling:DX` and `Fan:VariableVolume`
- `Coil:Cooling:DX` and `Fan:SystemModel` (cycling)
- `Coil:Cooling:DX:MultiSpeed` and `Fan:VariableVolume`
- `Coil:Cooling:DX:VariableSpeed` and `Fan:VariableVolume`